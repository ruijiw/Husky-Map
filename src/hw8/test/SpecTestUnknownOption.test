# Test the program has the correct output for unknown command
Unknown

# complete this test
q