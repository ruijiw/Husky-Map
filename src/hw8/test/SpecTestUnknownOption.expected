Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test the program has the correct output for unknown command
Unknown option

Enter an option ('m' to see the menu): 
# complete this test